versioned_docs/version-2.0/how_to_guides/evaluation/custom_code_auto_evaluator.mdx

```diff
@@ -0,0 +1,78 @@
+# Set up an Auto-Evaluator that runs Custom Python Code
+
+In this guide, we will walk through how to set up an auto evaluator that runs
+user defined python code over all new experiments on your dataset.
+
+## Set up the auto-evaluator
+
+Go to the datasets UI, select your dataset, and click the Auto Evaluators button. When prompted,
+select the `Custom Code` type evaluator.
+
+## Write your evaluation function
+
+:::note
+Custom Code evaluators use pyodide sandboxes under the hood to ensure security for our customers. You are
+restricted to import only their [supported packages](https://pyodide.org/en/stable/usage/packages-in-pyodide.html).
```

Comment:
FYI we don't allow all packages, only a few white listed ones

Reply:
These are ones we include:
```
  numpy: "numpy",
  pandas: "pandas",
  jsonschema: "jsonschema",
  scipy: "scipy",
  sklearn: "scikit-learn",
```

Reply:
Oh where do we document this?

Reply:
Sorry to clarify, we include the standard python libraries, but for pip packages those are the ones included.  I would list as:

* numpy
* pandas
* jsonschema
* scipy
* scikit-learn